stages/org.osbuild.selinux.meta.json

```diff
@@ -33,9 +33,15 @@
         }
       ],
       "properties": {
-        "file_contexts": {
+        "target": {
           "type": "string",
-          "description": "Path to the active SELinux policy's `file_contexts`"
+          "description": "Target path in the tree or on a mount",
```

Comment:
Should we mention that (again AFAICT) this only applied to file_contexts, exclude_paths but not to "labels" ?

Reply:
We could, but I'm not sure it's worth the CI spin. For labels you give it a full path and also the exact context so it's hard to think anything else could happen.

Reply:
or maybe we can make a deal.. :) feel free to push up a clarified description here if we can then just merge it immediately and not spin CI?

Reply:
It's fine, I understand we are in the land of diminishing return here :) Sorry for my nitpicks.